packager/CMakeLists.txt

```diff
@@ -64,4 +65,115 @@ add_subdirectory(third_party)
 add_subdirectory(tools)
 add_subdirectory(utils)
 add_subdirectory(version)
-add_subdirectory(app)
+
+set(libpackager_sources
+  app/job_manager.cc
+  app/job_manager.h
+  app/muxer_factory.cc
+  app/muxer_factory.h
+  app/libcrypto_threading.cc
+  app/libcrypto_threading.h
+  app/packager_util.cc
+  app/packager_util.h
+  app/single_thread_job_manager.cc
+  app/single_thread_job_manager.h
+  packager.cc
+  packager.h
+  )
+
+if(LIBPACKAGER_SHARED)
+  add_library(libpackager SHARED ${libpackager_sources})
+else()
+  add_library(libpackager STATIC ${libpackager_sources})
+endif()
+
+target_link_libraries(libpackager
+  file
+  hls_builder
+  media_chunking
+  media_codecs
+  media_crypto
+  demuxer
+  media_event
+  dvb
+  mp2t
+  mp4
+  packed_audio
+  ttml
+  formats_webm
+  wvm
+#  media_public
```

Comment:
What's up with this?

Reply:
I should have cleaned that up, there isn't any code to link against for media_public